web/wp-content/plugins/marquee/legacy/legacy.php

```diff
@@ -0,0 +1,241 @@
+<?php
```

Comment:
This code was not needed for CPT, the file is not included. Keeping it in the repo to revisit later.